panther_gazebo/include/panther_gazebo/gui/e_stop.hpp

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef PANTHER_GAZEBO_PLUGINS_ESTOP_HPP_
-#define PANTHER_GAZEBO_PLUGINS_ESTOP_HPP_
+#ifndef PANTHER_GAZEBO_E_STOP_HPP_
```

Comment:
```suggestion
#ifndef PANTHER_GAZEBO_GUI_E_STOP_HPP_
```
Please add folder name to the header guard.

Reply:
ok